site/content/tutorial/09-motion/02-spring/text.md

```diff
@@ -8,7 +8,7 @@ In this example we have two stores — one representing the circle's coordinates
 
 ```html
 <script>
-	import { spring } from 'svelte/motion';
+	import { spring } from "svelte/motion";
```

Comment:
huh, the Prettier config has `singleQuote: true` which is what we normally use. any idea why it's doing this?

Reply:
We didn't have prettier in this repo, @PuruVJ did you add it in another PR? 

It's likely inheriting from editor config/prettier default if this was done through the editor instead of a script.

Reply:
This was done through editor, and it's still sticking to single quote for some reason 🤷

Should I introduce a .prettierrc to the content folder? I could do it on root, but I'm not sure how it will affect the source code 🤔

So we could do this incrementally by having prettierrc only in content, or go the whole way by formatting everything in the repo. Also don't know how it will work with existing ESLint 🤔

Reply:
We should try to keep prettier changes scoped to the sites section for now (by placing a prettier config in there and opening the repo with that at the root, that way it's picked up but only for that subsection)

Reply:
What about the `site/content`? That's where the problem is arising 😅

Reply:
ok then add a `.prettierrc` to the root with the correct settings but also add a `.prettierignore` to tell it to not format anything outside `sites` and `site/content`

Reply:
Added a global .prettierrc, a prettierignore, and formatted all files with prettier CLI.

Also, changed the `html` code blocks to `svelte` just so prettier doesn't try auto formatting them